app/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React, {ButtonHTMLAttributes, memo} from 'react';
+import './Button.scss';
+
+function component(props: ButtonHTMLAttributes<HTMLButtonElement>): JSX.Element {
```

Comment:
Кажется, писал в одном из прошлых PR, что можно

```suggestion
function component({type = 'button', className = 'btn t-main}: ButtonHTMLAttributes<HTMLButtonElement>): JSX.Element {
```
и `defaultProps` не нужны

Вместо `ButtonHTMLAttributes` стоит объявить свой тип для пропсов, где явно описать все передаваемые атрибуты

Reply:
Да это можно, только я не добавлял этот файл, я просто переименовал каталог, не понимаю почему изменение поменилось как добавление а не переименование
Поправлю